build/publish/PublishContent.targets

```diff
@@ -38,4 +38,23 @@
       Overwrite="$(OverwriteOnPublish)" />
   </Target>
 
+  <!-- publishes checksums for the artifacts to blob storage in Azure -->
+  <Target Name="UploadChecksumsToAzure" DependsOnTargets="GenerateChecksums">
+    <Error Condition="'$(ChecksumContainerName)' == ''" Text="Missing property ChecksumContainerName." />
+    <Error Condition="'$(ChecksumCloudDropAccountName)' == ''" Text="Missing property ChecksumCloudDropAccountName." />
+    <Error Condition="'$(ChecksumCloudDropAccessToken)' == ''" Text="Missing property ChecksumCloudDropAccessToken." />
+    <!-- create the container if it doesn't exist -->
+    <CreateAzureContainer
+      AccountKey="$(ChecksumCloudDropAccessToken)"
+      AccountName="$(ChecksumCloudDropAccountName)"
+      ContainerName="$(ChecksumContainerName)" />
+    <!-- now upload the items -->
+    <UploadToAzure
+      AccountKey="$(ChecksumCloudDropAccessToken)"
+      AccountName="$(ChecksumCloudDropAccountName)"
+      ContainerName="$(ChecksumContainerName)"
+      Items="@(GenerateChecksumsInputs->'%(DestinationPath)')"
```

Comment:
Shouldn't this be consuming the GenerateChecksums Outputs rather than the inputs?